receiver/bindplaneauditlogs/logs.go

```diff
@@ -0,0 +1,226 @@
+	}, nil
+}
+
+func (r *bindplaneAuditLogsReceiver) Start(_ context.Context, _ component.Host) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	r.cancel = cancel
+	r.wg.Add(1)
+	go r.startPolling(ctx)
+	return nil
+}
+
+func (r *bindplaneAuditLogsReceiver) startPolling(ctx context.Context) {
+	defer r.wg.Done()
+	t := time.NewTicker(r.cfg.PollInterval)
+
+	err := r.poll(ctx)
+	if err != nil {
+		r.logger.Error("there was an error during the first poll", zap.Error(err))
+	}
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case <-t.C:
+			err := r.poll(ctx)
+			if err != nil {
+				r.logger.Error("there was an error during the poll", zap.Error(err))
+			}
+		}
+	}
+}
+
+func (r *bindplaneAuditLogsReceiver) poll(ctx context.Context) error {
+	logEvents := r.getLogs(ctx)
+	observedTime := pcommon.NewTimestampFromTime(time.Now())
+	logs := r.processLogEvents(observedTime, logEvents)
+	if logs.LogRecordCount() > 0 {
+		if err := r.consumer.ConsumeLogs(ctx, logs); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (r *bindplaneAuditLogsReceiver) getLogs(ctx context.Context) []AuditLogEvent {
+	var logs []AuditLogEvent
+
+	reqURL := r.cfg.BindplaneURL
+	reqURL = strings.TrimRight(reqURL, "/")
+	if !strings.Contains(reqURL, "://") {
+		reqURL = "http://" + reqURL
+	}
+	reqURL = reqURL + "/v1/audit-events"
+
+	req, err := http.NewRequestWithContext(ctx, "GET", reqURL, nil)
+	if err != nil {
+		r.logger.Error("error creating request", zap.Error(err))
+		return logs
+	}
+
+	query := req.URL.Query()
+	query.Add("since", r.lastTimestamp.Format(bindplaneTimeFormat))
+	req.URL.RawQuery = query.Encode()
+
+	req.Header.Add("X-Bindplane-Api-Key", r.cfg.APIKey)
+
+	res, err := r.client.Do(req)
+	if err != nil {
+		r.logger.Error("error making request", zap.Error(err))
+		return logs
+	}
+	defer res.Body.Close()
+
+	if res.StatusCode != http.StatusOK {
+		r.logger.Error("non-200 response", zap.String("status", res.Status))
+		return logs
+	}
+
+	body, err := io.ReadAll(res.Body)
+	if err != nil {
+		r.logger.Error("error reading response", zap.Error(err))
+		return logs
+	}
+
+	var response apiResponse
+	err = json.Unmarshal(body, &response)
+	if err != nil {
+		r.logger.Error("unable to unmarshal log events", zap.Error(err), zap.String("body", string(body)))
+		return logs
+	}
+
+	logs = response.AuditEvents
+
+	// Sort by timestamp (newest first)
+	sort.Slice(logs, func(i, j int) bool {
+		return logs[i].Timestamp.After(*logs[j].Timestamp)
+	})
+
+	// Update lastTimestamp
+	if len(logs) > 0 {
+		var latestTime time.Time
+		for _, event := range logs {
+			if event.Timestamp != nil && event.Timestamp.After(latestTime) {
+				latestTime = *event.Timestamp
+			}
+		}
+		if !latestTime.IsZero() {
+			r.lastTimestamp = latestTime
+		}
+	}
+
+	return logs
+}
+
+func (r *bindplaneAuditLogsReceiver) processLogEvents(observedTime pcommon.Timestamp, logEvents []AuditLogEvent) plog.Logs {
+	logs := plog.NewLogs()
```

Comment:
Can we include resource attributes for describing the Bindplane server? Something like `bindplane_url` or anything else that might be useful for filtering audit events based on the originating server.